ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/match_list_input.schema.yml

```diff
@@ -10,6 +10,36 @@ keys:
     type: dict
     display_name: Match Lists
     keys:
+      prefix_ipv4:
+        type: list
+        primary_key: name
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: Prefix-List Name
+            prefixes:
+              type: list
+              required: true
```

Comment:
You can consider enhancing the schema with min_length if you always want at least one entry. (we have received opposite feedback for IP ACLs where users want to define the ACL with no items it it, so they also did not want the key corresponding to `prefixes` to be required).